xds/internal/httpfilter/rbac/rbac.go

```diff
@@ -65,6 +65,7 @@ type builder struct {
 type config struct {
 	httpfilter.FilterConfig
 	config *rpb.RBAC
```

Comment:
Do we still need to save this if we have the engine?

Reply:
Yeah, we do. This config is used for certain pieces of logic that are xDS RBAC specific (i.e. not related to Ashitha's, so I would rather have it in rbac). These statements are regarding Rules "If absent, no enforcing RBAC policy will be applied", (this one applies to both of ours) and "At this time, if the RBAC.action is Action.LOG then the policy will be completely ignored, as if RBAC was not configurated." - A41 (this one is specific to mine).

Reply:
(See BuildServerInterceptor() for how this field is used.)

Reply:
In either of these instances, it seems as if we could leave the engine as `nil` instead - is that wrong?  Let's not carry around unnecessary state, unless it makes the code excessively complex.

Reply:
Yeah, good point. Deleted persisting config and made the conditional on whether config.chainEngine was nil in BuildServerInterceptor.